services/ui-src/src/types/users.ts

```diff
@@ -14,6 +14,7 @@ export interface MFPUser {
   family_name: string;
   full_name: string;
   state?: string;
+  userReports?: string[] | undefined;
```

Comment:
what was this added for? I don't see it in the UserProvider in the app so wondering if it's used outside of mocks

Reply:
I defined the `mockUsers` file based on what we have in [MCR](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/blob/eb88ec57c1af586b14b97cb46360ade71d65c9d9/services/ui-src/src/utils/testing/mockUsers.tsx#L4), which includes a `userReports` list, but given we aren't actually referencing that anywhere in the tests, I can remove it!